.golangci.yml

```diff
@@ -45,8 +45,8 @@ linters-settings:
 issues:
   include:
     - EXC0002
+  exclude-dirs:
+    - example
 
 run:
-  timeout: 8m
-  skip-dirs:
-    - example
+  timeout: 8m
```

Comment:
unexpected EOF. please remove this char. thanks

Reply:
good catch, updated, thanks!